src/app/projects/[projectId]/Analysis.tsx

```diff
@@ -27,13 +27,17 @@ interface AnalysisProps {
 interface ContractValues {
   name: string;
   description?: string | null;
-  startDate: string;
-  endDate: string;
-  status: string;
+  startDate?: string;
+  endDate?: string;
+  status?: string;
```

Comment:
why these undefined values? Also check the option of better use the drizzle type